src/app/component/core/header-new/header-new.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from '@angular/core';
+import {ModalService} from '../_modal/modal.service';
+import {MatDialog} from '@angular/material';
+import {FavoritePlaceComponent} from '../../map/favorite-place/favorite-place.component';
+import {ProposeCafeComponent} from '../propose-cafe/propose-cafe.component';
+import {FavoritePlaceService} from '../../../service/favorite-place/favorite-place.service';
+import {UserSettingComponent} from '../../user/user-setting/user-setting.component';
+import {Router} from '@angular/router';
+import {LocalStorageService} from '../../../service/localstorage/local-storage.service';
+import {JwtService} from '../../../service/jwt/jwt.service';
+import {UserService} from 'src/app/service/user/user.service';
+import {AchievementService} from 'src/app/service/achievement/achievement.service';
+import {HabitStatisticService} from 'src/app/service/habit-statistic/habit-statistic.service';
+import {filter} from 'rxjs/operators';
+import {LanguageService} from '../../../i18n/language.service';
+import {Language} from '../../../i18n/Language';
+
+
+@Component({
+  selector: 'app-header-new',
+  templateUrl: './header-new.component.html',
+  styleUrls: ['./header-new.component.scss']
+})
+export class HeaderNewComponent implements OnInit {
+  readonly notificationIcon = 'assets/img/notification-icon.png';
+  readonly userAvatar = 'assets/img/user-avatar.png';
+  readonly arrow = 'assets/img/arrow_grey.png';
+  dropdownVisible: boolean;
+  firstName: string;
+  userRole: string;
+  userId: number;
+  isLoggedIn: boolean;
+  language: string;
+
+  constructor(private modalService: ModalService,
+              public dialog: MatDialog,
+              private favoritePlaceService: FavoritePlaceService,
+              private localStorageService: LocalStorageService,
+              private jwtService: JwtService,
+              private router: Router,
+              private userService: UserService,
+              private achievementService: AchievementService,
+              private habitStatisticService: HabitStatisticService,
+              private languageService: LanguageService) {
+}
+
+  ngOnInit(): void {
```

Comment:
Please, delete the type. It is not necessary in hooks

Reply:
+